docs/source/api_reference.rst

```diff
@@ -28,17 +28,6 @@ Deep Feature Synthesis
     dfs
     get_valid_primitives
 
-Wrappers
-~~~~~~~~
-.. currentmodule:: featuretools
-
-scikit-learn (BETA)
--------------------
-.. autosummary::
-    :toctree: generated/
-
-    wrappers.DFSTransformer
```

Comment:
Removing the DFSTransformer from our docs as it only supports python 3.7 and 3.8 and is causing docs build failures